roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -214,6 +215,50 @@
     loop_var: vm_data
     index_var: family_id
 
+- name: Create VBMC entity
+  when:
+    - _vbmc_available is defined
+    - _vbmc_available | bool
+  vars:
+    cifmw_virtualbmc_machine: "cifmw-{{ item.key }}"
+    cifmw_virtualbmc_ipmi_port: >-
+      {{
+        cifmw_virtualbmc_ipmi_base_port + index
+      }}
+    cifmw_virtualbmc_action: "add"
+    _ipmi_host: >-
+      {%- if _cifmw_libvirt_manager_layout.vms.crc.target is defined -%}
+      {{_cifmw_libvirt_manager_layout_layout.vms.crc.target }}
+      {%- elif _cifmw_libvirt_manager_layout.vms.ocp.target is defined -%}
+      {{ _cifmw_libvirt_manager_layout.vms.ocp.target }}
+      {%- else -%}
+      {{ inventory_hostname }}
+      {%- endif -%}
+    cifmw_virtualbmc_ipmi_address: >-
+      {{ hostvars[_ipmi_host].ansible_default_ipv4.address }}
+  ansible.builtin.include_role:
+    name: virtualbmc
+    tasks_from: manage_host
+  loop: "{{ _libvirt_manager_networking.instances | dict2items }}"
+  loop_control:
+    index_var: index
+    label: "{{ item.key }}"
+
+- name: Create group inventories
```

Comment:
This should filter on the vms being `managed: false` - i.e don't add the ironic nodes that break the neutron whitebox tests?
Ref: https://github.com/openstack-k8s-operators/ci-framework/pull/1860

Reply:
added the `when` condition